package.json

```diff
@@ -9,28 +9,35 @@
   "scripts": {
     "build": "aegir build",
     "clean": "rm -rfv node_modules dist *.lock *-lock.json ",
-    "test": "aegir test",
+    "test": "aegir test --target browser",
```

Comment:
The example suggested targets for test, test:browser, and test:node.
But I suppose this project only targets a browser, right, so maybe we should only do that?

Reply:
Yes, this is because we don't intend to support node at the moment.